gufe/solventcomponent.py

```diff
@@ -69,15 +69,15 @@ def __init__(self, *,  # force kwarg usage
         self._negative_ion = negative_ion
 
         self._neutralize = neutralize
-        if ion_concentration is not None:
-            if (not isinstance(ion_concentration, unit.Quantity) or
-               not ion_concentration.is_compatible_with(unit.molar)):
-                raise ValueError(f"ion_concentration must be given in units of"
-                                 f" concentration, got {ion_concentration}")
-            # concentration requires both ions be given
-            if ion_concentration > 0:
-                if self._negative_ion is None or self._positive_ion is None:
-                    raise ValueError("Ions must be given for concentration")
+
+        if (not isinstance(ion_concentration, unit.Quantity)
+           or not ion_concentration.is_compatible_with(unit.molar)):
```

Comment:
just readability, but I think the `or` should be moved over one space to align *after* the paren.

Actually, for something like this, I'm starting to prefer the style black uses for long if statements (which makes a clearer separation of the block following the if):

```python
if (
    thing
    or thing2
):
    do_something()
```